test/built-ins/Function/15.3.5.4_2-12gs.js

```diff
@@ -7,6 +7,7 @@ description: >
     Strict mode - checking access to non-strict function caller from
     non-strict function (eval includes strict directive prologue)
 flags: [noStrict]
+features: [caller]
```

Comment:
The other tests work without any changes because they
- either have been updated in the past to work in engines without `caller` (https://github.com/tc39/test262/blob/master/test/built-ins/Function/15.3.5.4_2-16gs.js#L23)
- or the default `caller` property from `Function.prototype` (8.2.2 CreateIntrinsics, step 12) gives the correct result (that means a `TypeError` is thrown).